app/src/androidTest/java/com/github/se/cyrcle/ui/card/CardScreenTest.kt

```diff
@@ -90,7 +90,9 @@ class CardScreenTest {
     parkingViewModel.selectParking(TestInstancesParking.parking1)
     composeTestRule.setContent { CardScreen(navigationActions, parkingViewModel, userViewModel) }
 
-    composeTestRule.onNodeWithTag("TopAppBarTitle").assertTextContains("Description of Parking")
+    composeTestRule
+        .onNodeWithTag("TopAppBarTitle")
+        .assertTextContains("Description of Unnamed Parking")
     composeTestRule.onNodeWithTag("ParkingImagesRow").onChildren().assertCountEquals(1)
     composeTestRule.onNodeWithTag("CapacityColumn").onChildAt(1).assertTextContains("51-100 spots")
```

Comment:
This string must be refactored as well I think.

Reply:
I will try to replace `"51-100 spots"` by calling the appropriate `enum` in the next commit, instead of relying of the hardcoded one.